php/Terminus/Auth.php

```diff
@@ -36,12 +36,58 @@ public static function loggedIn() {
   }
 
   /**
+   * Execute the login based on a new refresh token
+   *
+   * @param [string] $token Refresh token to initiate login with
+   * @return [boolean] True if login succeeded
+   */
+  public function logInViaRefreshToken($token) {
+    $options = array(
+      'headers' => array('Content-type' => 'application/json'),
+      'cookies' => array('Authorization: Bearer' => $token),
```

Comment:
Actually I think we should send the refresh token as part of the body. I'm not sure it's quite acting as a bearer token in this case. 


Once we have a JWT, that should be sent using an Authorization header:

    $options = array(
        'headers' => array(
            'Content-type' => 'application/json',
            'Authorization' => 'Bearer ' . $jwt_token 
        ),


So that the header ends up being:

`Authorization: Bearer XXXXXXXXXXXXX`